### ISSUE TITLE
Bianca Bracchetti - CSS Project

### ISSUE THREAD

**Issue by bxb760** [b-ivey/css-project#4]
https://github.com/bxb760/CSS-project

**Comment by b-ivey** [comment]
Love it!! Love the purple -- that's my favorite part. 

Small, tiny issues:

1. Add some padding between the photo and your contact info.

2. Make sure you have no spaces in the names of your file. Your main file is labeled CV .html -- that could break a real site.

3. Make sure all the file names are lowercase, meaning no UPPERCASE letters at all.

Awesome use of the grids. You really put a lot of thought into this and it shows. Can't wait to see your main project.

**Comment by bxb760** [comment]
Dear professor,

I'm doing my final project. I wanted to ask you if for every html page we needed a CSS file or do we need to use the same file for everything?

Best
Bianca
________________________________
Da: b-ivey <notifications@github.com>
Inviato: mercoledì 15 aprile 2020 05:36
A: b-ivey/css-project <css-project@noreply.github.com>
Cc: Bracchetti, Bianca <bxb760@miami.edu>; Author <author@noreply.github.com>
Oggetto: Re: [b-ivey/css-project] Bianca Bracchetti - CSS Project (#4)


Love it!! Love the purple -- that's my favorite part.

Small, tiny issues:

  1.  Add some padding between the photo and your contact info.

  2.  Make sure you have no spaces in the names of your file. Your main file is labeled CV .html -- that could break a real site.

  3.  Make sure all the file names are lowercase, meaning no UPPERCASE letters at all.

Awesome use of the grids. You really put a lot of thought into this and it shows. Can't wait to see your main project.

**Comment by b-ivey** [comment]
One css file should work for all the pages. Be sure to keep to the same grid layout on all pages

**Comment by bxb760** [comment]
But if I want two different grids I can do it right? (one for the first page and the second one for all the inside page)
________________________________
Da: b-ivey <notifications@github.com>
Inviato: giovedì 16 aprile 2020 16:38
A: b-ivey/css-project <css-project@noreply.github.com>
Cc: Bracchetti, Bianca <bxb760@miami.edu>; Author <author@noreply.github.com>
Oggetto: Re: [b-ivey/css-project] Bianca Bracchetti - CSS Project (#4)


One css file should work for all the pages. Be sure to keep to the same grid layout on all pages

**Comment by b-ivey** [comment]
Yes, you could. You’d have to either do separate CSS files OR name your groups differently and apply the grid codes in those <div> tags. It’s standard practice to usually use one CSS file.
